oteapi/models/genericconfig.py

```diff
@@ -113,36 +134,16 @@ def popitem(self) -> "Tuple[str, Any]":
             This is due to the fact that attributes are stored in a random order when
             initializing the model.
 
-            However, it will respect LIFO with respect to the internal `__dict__`.
+            However, it will respect LIFO with respect to the internal `model_fields`.
 
         """
         if not self:
             raise KeyError(f"popitem(): {self.__class__.__name__} is empty")
 
-        key = list(self.__dict__)[-1]
+        key = list(self.keys())[-1]
         value = self.pop(key)
         return key, value
 
-    class Config:
```

Comment:
make sure these doc strings are transferred to where they occur in the new version